workers/unity/Packages/com.improbable.gdk.mobile/Utility/LaunchArguments.cs

```diff
@@ -30,7 +32,25 @@ public static class LaunchArguments
                 UnityEngine.Debug.LogException(e);
             }
 
-            return new Dictionary<string, string>();
+            return null;
+        }
+
+        public static Dictionary<string, string> GetiOSArguments()
+        {
+            try
+            {
+                var argumentsEnvVar = System.Environment.GetEnvironmentVariable(LaunchArgumentsEnvKey);
+                if (argumentsEnvVar != null)
+                {
+                    return CommandLineUtility.ParseCommandLineArgs(argumentsEnvVar.Split(' '));
```

Comment:
What are the odds of having a quoted string with spaces as part of the arguments? Like `--path "Applications\Probably a bad idea"`?

Reply:
Right now it's a non case - we control what is passed here as environment variable and we only pass ip address and the key for the argument. We'll have to improve this in the future once there is a chance we pass strings with spaces